lib/desired-caps.js

```diff
@@ -5,6 +5,9 @@ const desiredCapConstraints = {
   browserVersion: {
     isString: true
   },
+  acceptInsecureCerts: {
```

Comment:
I assume this is redundant because BaseDriver already defines such cap

Reply:
I thought so, but if it has not been defined, then an warning message below was printed
```
[SafariDriver@41f9] The following capabilities were provided, but are not recognized by Appium:
[SafariDriver@41f9]   acceptInsecureCerts
```